components/Bounties/index.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import Header from './header';
+import Card from './card';
+import StyledCards from './StyledCards';
+import { bountiesDataTmp } from '../../constants';
+import StyledSpinner from './loading/StyledSpinner';
+import StyledLoadingBounties from './loading/StyledLoadingBounties';
+import Spin from '../Spin';
+
+const Bounties = ({styling, loading}) => (
+  <div>
+    {loading &&
```

Comment:
You can do `loading ? <Component1> : <Component2>`

Reply:
indeed